### ISSUE TITLE
508-defect-2 [SCREENREADER, COGNITION]: Purpose of a link MUST be clear in link text

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#18418]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Under more information, the link "For state agencies and organizations that use VA forms" does not provide a clear purpose on its own. Screen readers may be unsure if they should follow the link when the user agent provides a list of all the links on a page.

For sighted users it is clear that the Directions link is part of the search result. For non-sighted users hearing the search results, visual design isn't available to support comprehension, so it is added using `aria-label` attributes or screen reader only text.

## Point of Contact

**VFS Point of Contact:** Josh

## Acceptance Criteria

- [ ] As I screen reader user, I want to explicitly know the purpose of the link on a page.

## Environment

* Operating System: all
* Browser: all
* Screenreading device: all
* Server destination: production

## Solution

Provide the purpose of the link within the link text. For example, "Get the latest VA form updates"

## WCAG or Vendor Guidance (optional)

* [Ensure that links with the same accessible name serve a similar purpose, Deque University](https://dequeuniversity.com/rules/axe/3.5/identical-links-same-purpose?application=AxeChrome)

## Screenshots
<img width="746" alt="Screen Shot 2021-01-12 at 2 49 57 PM" src="https://user-images.githubusercontent.com/14154792/104365410-f5063200-54e5-11eb-9a7e-13acc04a3123.png">

**Comment by zacharymorel** [comment]
@joshkimux Above we'd be shooting for something like aria-label="Go see {{title of section here}}", correct?

**Comment by joshkimux** [comment]
@zacharymorel Ideally if the designers and content folks are OK with it, it would be better to opt to change the string itself to avoid having to use aria. So e.g. "Get the latest VA form updates" as a heading link would be better than "For state agencies and organizations that use VA forms"

**Comment by zacharymorel** [comment]
Agreed here! I think that falls on the content team for the node in Drupal? 
@MarciMcGuireGCIO @johnhashva Do ya'll no more about that?

**Comment by zacharymorel** [comment]
From the Drupal Meta data:
```
  layout: page.drupal.liquid
  path: find-forms
  entityId: 2352
```

So I believe the entity ID is 2352 for the Page Node.

**Comment by johnhashva** [comment]
@zacharymorel You'll need to Zoom walk me through this so I can understand it (unless @MarciMcGuireGCIO already does, that is).

**Comment by MarciMcGuireGCIO** [comment]
@sshein -- if an a11y issue originates in Drupal, do I need to get with CMS a resolution?  Thx!

**Comment by MarciMcGuireGCIO** [comment]
> Agreed here! I think that falls on the content team for the node in Drupal?
> @MarciMcGuireGCIO @johnhashva Do ya'll know more about that/ make the change yourself in. Drupal?

@zacharymorel cc @johnhashva I'm trying to find out how we handle.

**Comment by johnhashva** [comment]
@MarciMcGuireGCIO talk me through this tomorrow and I can help figure out (a) if we really want to change this and (b) if yes, how.

**Comment by jenniferlee-dsva** [comment]
Yep - this is a content update in Drupal. I worked with Danielle Thierry and Mark Greenberg, the Lighthouse form API PM at the time on the text here. Agree with @joshkimux suggestion above -- only tricky thing here is that this is NOT for Veterans. 

Veterans are not going to use the API.  The API is primarily for states, VSOs, and other organizations that consume VA forms. We og did not include a link to the API here bc it's not for Veterans, but we were soon getting pinged by different state agencies (like New Jersey) and VSO orgs about how to scrape the full list of VA forms data.  

Let me noodle on this with Danielle (and Mark) - because we don't want to mislead Veterans into thinking that the API is something meant for them.

**Comment by MarciMcGuireGCIO** [comment]
Thx @jenniferlee-dsva !

**Comment by jenniferlee-dsva** [comment]
**[VA forms API for states and Veteran organizations](https://developer.va.gov/explore/vaForms/docs/vaForms?version=current )**
Automatically get VA form updates with our forms API. The API is free and lets you filter by form number, name, and other form data.

I think this tweak addresses the screen reader issue, while still making it clear who this is for. Lmk if this looks good, and I'm happy to make the Drupal change - it's like 5 minutes. @MarciMcGuireGCIO @johnhashva

**Comment by johnhashva** [comment]
@jenniferlee-dsva works for me!

**Comment by MarciMcGuireGCIO** [comment]
👌🏻 👨🏻‍🍳  Perfect @jenniferlee-dsva !

**Comment by jenniferlee-dsva** [comment]
Preview - still in Draft mode: http://preview-prod.vfs.va.gov/preview?nodeId=2352

![image](https://user-images.githubusercontent.com/43381063/104762558-8ff34c00-5732-11eb-8d8d-12b44eb9971a.png)

FYSA - @markgreenburg - we're making the above text tweaks to the API text link, so it's clearer for accessiblity/screenreaders. Just wanted to make sure you had a chance to preview this before it goes live.

**Comment by jenniferlee-dsva** [comment]
Per Slack, Mark doesn't see any problems with updated text, so this is now set to publish with the next content deployment. Feel free to close this ticket. @MarciMcGuireGCIO

**Comment by MarciMcGuireGCIO** [comment]
Thank you @jenniferlee-dsva .
